ui/src/main/java/com/nextcloud/android/common/ui/color/ColorUtil.kt

```diff
@@ -24,6 +24,7 @@ package com.nextcloud.android.common.ui.color
 
 import android.content.Context
 import android.graphics.Color
+import android.util.Log
```

Comment:
At some point I'd like to move the `Log_OC` stack from android-library into this repo, but for now `Log` must do.

Reply:
👍